### PR TITLE
Updating the doc for browser usage report

### PR DESCRIPTION
**Pull request by sarahwxy**
Adding a section for browser action flow so global admins know what is this feature for.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/activity-reports/browser-usage-report.md [hunk 1]
```diff
@@ -32,6 +32,14 @@ The Microsoft 365 **Reports** dashboard shows you an activity overview across th
 1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page. 
 2. From the dashboard homepage, click on the **View more** button on the Microsoft browser usage card.
 
+## How to notify users to upgrade their browser
+
+![Microsoft browser usage report action flow](../../media/1ef4eb08-18b8-4dda-aa15-1aad013ecd70.png)
+
+Global admins can opt-in to sending messages to users, who are using Microsoft 365 services on Edge Legacy (unsupported) and Internet Explorer (soon to be unsupported). This targeted message notifies users that support for these browsers will send soon and links to a support article with information on Microsoft Edge and simple steps to follow to switch browsers.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/browser-usage-report.md:39]:
```suggestion
Global admins can opt in to sending messages to users who are using Microsoft 365 services on Edge Legacy (unsupported) and Internet Explorer (soon to be unsupported). This targeted message notifies users that support for these browsers will end soon and links to a support article with information on Microsoft Edge and simple steps to follow to switch browsers. 
```

File: microsoft-365/admin/activity-reports/browser-usage-report.md [hunk 2]
```diff
@@ -32,6 +32,14 @@ The Microsoft 365 **Reports** dashboard shows you an activity overview across th
 1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page. 
 2. From the dashboard homepage, click on the **View more** button on the Microsoft browser usage card.
 
+## How to notify users to upgrade their browser
+
+![Microsoft browser usage report action flow](../../media/1ef4eb08-18b8-4dda-aa15-1aad013ecd70.png)
+
+Global admins can opt-in to sending messages to users, who are using Microsoft 365 services on Edge Legacy (unsupported) and Internet Explorer (soon to be unsupported). This targeted message notifies users that support for these browsers will send soon and links to a support article with information on Microsoft Edge and simple steps to follow to switch browsers. 
+
+You can find this feature on the report page. Once the message is created, users will get notified at the frequency specified until August 17, 2021. You can turn off this feature at any time to stop sending notifications to users. To begin sending notifications again, turn the feature back on.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/browser-usage-report.md:41]:
```suggestion
You can find this feature on the reports page. Once the message is created, users will get notified at the frequency specified until August 17, 2021. You can turn off this feature at any time to stop sending notifications to users. To begin sending notifications again, turn the feature back on.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @sarahwxy, kindly commit the suggested minor changes if you agree with them. Many thanks.

**Comment by yogkumgit** [comment]
Gentle reminder @sarahwxy

**Comment by yogkumgit** [comment]
@sarahwxy Gentle reminder, please update the document with suggested changes.

**Pull request merged**